modules/carto/src/api/maps-api-client.js

```diff
@@ -1,76 +1,67 @@
-import {getDefaultCredentials} from '../auth';
+import {getDefaultCredentials, getMapsVersion} from '../config';
 
 const DEFAULT_USER_COMPONENT_IN_URL = '{user}';
-const REQUEST_GET_MAX_URL_LENGTH = 2048;
+const DEFAULT_REGION_COMPONENT_IN_URL = '{region}';
```

Comment:
Ok, I see the new param. 
One question: isn't the module going to use the MAX_URL threshold to use a GET vs a POST request?

Reply:
Good point! We're deprecating the POST in vector because they're not cached in the CDN and the config is far smaller since we're removing all the stuff of raster (like cartocss, dataviews, etc...)